doc/usage/configuration.rst

```diff
@@ -756,9 +756,15 @@ documentation on :ref:`intl` for details.
    If true, a document's text domain is its docname if it is a top-level
    project file and its very base directory otherwise.
 
+   If set to string, all document's text domain is this string, making all
+   documents use single text domain.
+
    By default, the document ``markup/code.rst`` ends up in the ``markup`` text
    domain.  With this option set to ``False``, it is ``markup/code``.
 
+   .. versionchanged:: 3.2
```

Comment:
We'll release 3.2 in this weekend. But some discussion is still needed for this PR. So this would be a part of 3.3.